.github/workflows/unitctl.yml

```diff
@@ -186,4 +186,19 @@ jobs:
         with:
           artifacts: "unitctl-*"
           tag: ${{github.event_name == 'workflow_dispatch' && inputs.version}}
+          name: unitctl-${{(github.event_name=='workflow_dispatch' && inputs.version) || github.ref_name}}
```

Comment:
A part of me really wants to namespace our tags and identifiers with `unitctl/...` instead of `unitctl-...`

Do you have strong feelings either way?

Reply:
I dont have strong feelings about it!